### PR TITLE
 Addition of M24 & Cheek pad

### PR DESCRIPTION
**Pull request by J-Cardona**
I would love to see this weapon in the game, it's a bolt-action sniper rifle known because it's the adaptation of  a remington .700 but modified for the police/paramilitary purposes.

https://en.wikipedia.org/wiki/M24_Sniper_Weapon_System

As well as the cheek pad a nice stock modification for a vast variety of sniper rifles and DMRs.

Sorry if my english ain't great , though I hope everything is understandable.

This is my first project and I'm thrilled to do many more! :D

### CODE REVIEW

**Comment by ymber** [comment]
An M24 should probably have a default_mods array for optics and such.

File: data/json/items/gunmod/stock.json [hunk 1]
```diff
@@ -69,5 +69,25 @@
     "mod_targets": [ "smg", "rifle", "shotgun", "launcher" ],
     "handling_modifier": 4,
     "min_skills": [ [ "weapon", 2 ] ]
+  },
+  {
+    "id": "cheek_pad",
+    "type": "GUNMOD",
+    "name": "cheek pad",
+    "description": "An add on stock for sniper rifles and even DMRs considerably improving eye relief efficiency, prevents slipping, and reduces recoil to the neck and cheek.",
```
Comment (ymber) [inline data/json/items/gunmod/stock.json]: I think "add on stock" here should be "stock accessory".

**Comment by Jownky** [comment]
Thanks for the correction! :D 

By the way how do I write  the 'default_mods' for optics ? I'm a little bit new here

**Comment by ymber** [comment]
`"default_mods": [ "bipod", "rifle_scope", "cheek_pad" ]`

Get a look at how it's handled in the m107a1 in data/json/items/gun/50.json.

**Comment by Jownky** [comment]
Is it like the mods that it already comes with when you find it?

**Comment by ymber** [comment]
`default_mods` defines what mods a weapon spawns with. Normally you use it for things a weapon comes with from the factory or a standard military configuration. There's also `built_in_mods` for things that we handle as gunmods in the game but are really integral parts of the weapon.

**Comment by Jownky** [comment]
Thanks for the info! :D 
So what could be an example for a built_in_mod?

**Comment by ymber** [comment]
Other barrels in combination guns, the folding stock on a SIG 550, the second magazine in a KSG. Stuff that isn't actually modular but we handle internally as gunmods.

**Pull request merged**